config/proto/targets.proto

```diff
@@ -0,0 +1,46 @@
+/*
+  This proto defines the config for a Target. 
+  Targets are used by Hermes and Cloudprober to identify storage systems to monitor.
+*/
+syntax = "proto3";
+
+package hermes;
+
+option go_package = "github.com/googleinterns/step224-2020/targets/proto";
+
+// TargetsDef contains all of the metadata necessary for Hermes to establish a connection to a storage system.
+// Every probe request will require one or more targets.
+message TargetsDef {
+  enum TargetSystem {
```

Comment:
```suggestion
  // TargetSystem is expected to be a storage system.
  // It will stay constant until support for new storage systems is added.
  enum TargetSystem {
```